instaloader/instaloader.py

```diff
@@ -116,9 +116,18 @@ def __init__(self,
                  post_metadata_txt_pattern: str = None,
                  storyitem_metadata_txt_pattern: str = None,
                  graphql_rate_limit: Optional[int] = None,
-                 max_connection_attempts: int = 3):
-
-        self.context = InstaloaderContext(sleep, quiet, user_agent, graphql_rate_limit, max_connection_attempts)
+                 max_connection_attempts: int = 3,
+                 proxy: Optional[str] = None,
+                 ):
```

Comment:
The code formatting is not consistent with the rest of Instaoader (especially lines that only contain `)` or `):`, and lines ending with `(`)

(affects several changes in this pull request)